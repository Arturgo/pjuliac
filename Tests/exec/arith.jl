
println(-42)
println(40+2)
println(44-2)
println(6*7)
println(div(84,9))
println(div(-84,9))
println(div(84,-9))
println(div(-84,-9))
println(84 % 9)
println(-84 % 9)
println(84 % -9)
println(-84 % -9)

