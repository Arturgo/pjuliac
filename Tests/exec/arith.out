-42
42
42
42
9
-9
-9
9
3
-3
3
-3
